### PR TITLE
Finished Commits R' Us transcript

### PR DESCRIPTION
**Pull request by emilylwan**
## Pull Request Summary

New example transcript to upload into our database.

### Change types

- [x] Non-breaking change
- [x] Adding tests

### Testing
- [ x] I have added testing to this PR 

### How to manually test this PR (QA instructions)
You can upload the transcript or access with project id: b3fpk8.

### CODE REVIEW

File: examples/commitsRUsPlaceTrackOrder.json [hunk 1]
```diff
@@ -0,0 +1,1070 @@
...
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We couldn't find your order. Please re-enter your order number.\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We couldn't find your order. Please re-enter your order number.\"}}"
+    },
+    {},
```
Comment (WeissSa) [inline examples/commitsRUsPlaceTrackOrder.json:656]: Nice!

**Review by WeissSa — APPROVED** [review]
LGTM :+1:  (took a quick scroll-through)

**Review by KayleeLChan — APPROVED** [review]
Add her to the list lesgooo 💃 💃 💃 Our data are dummies but we are not 🤓

File: examples/commitsRUsPlaceTrackOrder.json [hunk 2]
```diff
@@ -0,0 +1,1070 @@
...
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"We have found your order for Add Chatbot pull request. What would you like to know?\"}}"
+    },
+    {
+      "project_id":"b3fpk8",
+      "trace_type":"text",
+      "trace_payload":"{\"payload\":{\"message\":\"Sorry we can't help you with that. Please contact us at 416-555-5555.\"}}"
```
Comment (KayleeLChan) [inline examples/commitsRUsPlaceTrackOrder.json:989]:
```suggestion
      "trace_payload":"{\"payload\":{\"message\":\"Sorry we can't help you with that. Please contact us at 416-bes-tflo.\"}}"
```
Just a suggestion for all the phone number endings LOL

**Pull request merged**